### ISSUE TITLE
runtime_environment_variables are not passed to App Runner

### ISSUE THREAD

**Issue by pawelszczerbicki** [hashicorp/terraform-provider-aws#19561]
### Expected Behavior
runtime_environment_variables should be passed to app runner

### Actual Behavior

Using this configuration variable `test` is not passed to app runner. I checked using NodeJs simple app image printing all variables passed to service `console.log(process.env)`

Creating App Runner from AWS console and adding variables works correctly

 ```
  source_configuration {
    authentication_configuration {
      access_role_arn = var.role_arn
    }
    image_repository {
      image_configuration {
        port = var.port
        runtime_environment_variables = {
          "test" = "xxx"
        }
      }
      image_identifier = var.image
      image_repository_type = var.repository_type
    }
  }
```

**Comment by timotas** [comment]
Yes, I am seeing this also. Specifying the variables in Terraform do not pass them to the service. Confirmed by `aws apprunner describe-service ...`. Instead when specifying the variables during service creation in the web console, they are visible in the CLI.

**Comment by timotas** [comment]
This is duplicate of https://github.com/hashicorp/terraform-provider-aws/issues/19469 and is fixed in https://github.com/hashicorp/terraform-provider-aws/pull/19471 which is merged already, and is scheduled for 3.43.0 release.
